src/wp-content/themes/twentyseventeen/inc/block-patterns.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * Twenty Twenty Theme: Block Patterns
+ *
+ * @package WordPress
+ * @subpackage Twenty_Seventeen
+ * @since Twenty Seventeen 2.4
+ */
+
+/**
+ * Register Block Pattern Category.
+ */
+if ( function_exists( 'register_block_pattern_category' ) ) {
+
+	register_block_pattern_category(
+		'twentyseventeen',
+		array( 'label' => __( 'Twenty Seventeen', 'twentyseventeen' ) )
+	);
+}
+
+/**
+ * Register Block Patterns.
+ */
+if ( function_exists( 'register_block_pattern' ) ) {
+	register_block_pattern(
+		'twentyseventeen/large-heading-with-button',
+		array(
+			'title'      => __( 'Large Heading with Button', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:heading {"level":1,"textColor":"black","style":{"typography":{"fontSize":50}}} -->
+            <h1 class="has-black-color has-text-color" style="font-size:50px">' . __( 'We can put together marketing campaigns and attract leads through search engine optimization.', 'twentyseventeen' ) . '</h1>
```

Comment:
Do these strings need to be translated _and_ escaped? I'd been escaping them for other themes just to be safe, but wasn't sure.

Reply:
This came up in 2021 https://github.com/WordPress/twentytwentyone/issues/479 and I'm not sure the final answer — for the older themes I think the standard is to not escape, [translations are considered trusted](https://core.trac.wordpress.org/ticket/30724) (so… maybe we need to remove escaping from 2020 & 2019 too).